cekit/descriptor/module.py

```diff
@@ -24,7 +24,8 @@ def __init__(self, descriptor, path, artifact_dir):
         self.skip_merging = ['description',
                              'version',
                              'name',
-                             'release']
+                             'release',
+                             'help']
```

Comment:
@rnc This PR additionally adds a change that was missed in https://github.com/cekit/cekit/pull/445

Reply:
Ok